src/api/main.py

```diff
@@ -0,0 +1,65 @@
+import os
+from os.path import join
+import errno
+import shutil
+from subprocess import call
+import json
+import uuid
+
+from flask import Flask, jsonify
+
+from rwd.Rapid_Watershed_Delineation import Point_Watershed_Function
+
+app = Flask(__name__)
+
+
+@app.route('/rwd/<lat>/<lon>', methods=['GET'])
+def run_rwd(lat, lon):
```

Comment:
Based on @dtarb's comments, I think we should move the two snapping vars to arguments here, and have them default to 1 & 10000, then we can work on finding a UI spot to generate values for them later.  The `np` var should probably just be a constant for now with the idea that it may be loaded from an env setting based on which environment the script is loaded in.